doc/concepts/hardware.rst

```diff
@@ -3,31 +3,39 @@
 Hardware context
```

Comment:
Combining reformatting with changes makes spotting the change difficult. Rather do this in separate changesets.